### PR TITLE
feat: allow setting up federation via `fedimint-cli`

### PR DESCRIPTION
**Pull request by douglaz**
Port of https://github.com/fedimint/fedimint/pull/3796

Plus missing methods and minor improvements

Closes https://github.com/fedimint/fedimint/issues/3795

In future we may create another command that does the full setup using these subcommands and use them to setup devimint, perhaps replacing the current method: https://github.com/fedimint/fedimint/blob/0b8cdd855cc46e58a74fa04678849bddd8da3832/devimint/src/federation.rs#L501

Calling example:
```bash
fedimint-cli admin dkg --ws ws://127.0.0.1:24329/ ws-status
```

### CODE REVIEW

File: fedimint-cli/src/lib.rs [hunk 1]
```diff
@@ -694,6 +733,83 @@ impl FedimintCli {
                         .map_err_cli_msg("invalid response")?,
                 ))
             }
+            Command::Admin(AdminCmd::WsStatus) => {
```
Comment (dpc) [inline fedimint-cli/src/lib.rs:736]: Thought: We should split handling admin commands to a separate function. The whole thing is getting way too long.

**Review by dpc — APPROVED** [review]

File: fedimint-cli/src/lib.rs (hunk 1 above)
Comment (douglaz) [reply fedimint-cli/src/lib.rs]: Ok, reorganized a bit now it's a subcommand and calls another function.

**Comment by elsirion** [comment]
Should have a devimint test to see if it actually works :P

**Review by dpc — APPROVED** [review]

**Review by elsirion — APPROVED** [review]
Just be aware this is very likely to break (or not even work in the first place) without testing. But since it's just some dev tooling let's get it in and iterate.

**Comment by fedimint-backports** [comment]
Successfully created backport PR for `releases/v0.3`:
- #4713

**Pull request merged**